### DISCUSSION TITLE
predicted value for random survival forest

### DISCUSSION THREAD

**Discussion by mao223** [kogalur/randomForestSRC#270] [Q&A]
Good day,

Could I get clarification what exactly the predicted values are for a random survival forest? For example:
```
predict(rsf_obj, newdata = newdata)$predicted
```
Thank you for a wonderful package and I deeply appreciate your help this far.

**Comment by ishwaran** [comment]
The value returned by `o$predicted.oob` (and `o$predicted)` is specific to the family that was fit, whether it was regression, survival classification and so forth.  

See [this section from getting started](https://www.randomforestsrc.org/articles/getstarted.html#prediction-and-terminal-node-statistics-1) for details on exactly what is returned for each family.

**Reply by mao223** [reply]
Thank you!!!
